src/02-common-principles.md

```diff
@@ -644,6 +540,147 @@ for more information.
 }
 ```
 
+## The Inheritance Principle
+
+1.  Any metadata file (such as `.json`, `.bvec` or `.tsv`) MAY be defined at any directory level.
+
+1.  For a given data file, any metadata file is applicable to that data file if:
+    1.  It is stored at the same directory level or higher;
+    1.  The metadata and the data filenames possess the same suffix;
+    1.  The metadata filename does not include any entity absent from the data filename.
+
+1.  A metadata file MUST NOT have a file name that would be otherwise applicable
```

Comment:
```suggestion
1.  A metadata file MUST NOT have a filename that would be otherwise applicable
```
if there would be agreement on https://github.com/bids-standard/bids-specification/pull/962 prompted by this line ;-)